src/main-enhancer.js

```diff
@@ -91,12 +102,12 @@ module.exports = overrides => storeCreator => (reducer, initialState) => {
       isDispatching: false,
       isUpdating: false,
       forwardOnUpdate: true,
-      senderClientId: null,
+      senderClientId: null
```

Comment:
```suggestion
      senderClientId: null,
```

Same thing as above ^ I prefer trailing comma